lib/internals.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This module is intended for custom handler developers and exposes internal
+ * API and utilities. No public documentation is provided.
+ */
+
+import Logger from './Logger';
+import EnhancedEventEmitter from './EnhancedEventEmitter';
+import * as utils from './utils';
+import * as ortc from './ortc';
+import * as sdpCommonUtils from './handlers/sdp/commonUtils';
+import * as sdpUnifiedPlanUtils from './handlers/sdp/unifiedPlanUtils';
+import * as sdpPlanBUtils from './handlers/sdp/planBUtils';
+import * as sdpPlainRtpUtils from './handlers/sdp/plainRtpUtils';
+import RemoteUnifiedPlanSdp from './handlers/sdp/RemoteUnifiedPlanSdp';
+import RemotePlanBSdp from './handlers/sdp/RemotePlanBSdp';
+import RemotePlainRtpSdp from './handlers/sdp/RemotePlainRtpSdp';
+
+export {
+	Logger,
+	EnhancedEventEmitter,
+	utils,
+	ortc,
+	sdpCommonUtils,
+	sdpUnifiedPlanUtils,
+	sdpPlainRtpUtils,
+	sdpPlanBUtils,
```

Comment:
Can we plz place sdpPlanBUtils before sdpPlainRtpUtils? (this was probably my fault)

Reply:
sure, done